sdk/lib/3rdparty/zlib/contrib/minizip/iowin32.c

```diff
@@ -95,7 +95,6 @@ static voidpf win32_build_iowin(HANDLE hFile)
 
 voidpf ZCALLBACK win32_open64_file_func (voidpf opaque,const void* filename,int mode)
 {
-    //const char* mode_fopen = NULL;
```

Comment:
Please revert.

Reply:
right, will comment out (as before) instead of removing line